build.sbt

```diff
@@ -7,11 +7,12 @@ lazy val `cats-time` = project
   .disablePlugins(MimaPlugin)
   .enablePlugins(NoPublishPlugin)
   .settings(commonSettings)
-  .aggregate(core.jvm, core.jvm)
+  .aggregate(core.jvm, core.js, testKit.jvm, testKit.js)
 
 lazy val core = crossProject(JSPlatform, JVMPlatform)
   .crossType(CrossType.Pure)
   .in(file("modules/core"))
+  .dependsOn(testKit % "test->compile")
```

Comment:
The idea with a `testkit` is to avoid the "test->compile" type of dependency. I think this could be removed considering the file you moved

Reply:
Yes, we can definitely omit the explicit scope definition here. 

My point was to narrow down scopes to the minimum possible variant. 
If we omit explicit scope sbt will consider it as "compile->compile", but apparently nothing in `core` module in `src/main/scala` will ever depend on arbitraties (only `core` tests require it). 

Shall we omit it?

Reply:
Hmm. does this mean we can really omit the `dependsOn`? It probably makes sense in case downstream you want to use the instances but don't care about the arbitraries

Reply:
Unfortunately, no. We need to define the `core->testkit` dependency so that tests in `core` could import arbitrariness.
Anyway, for the sake of clarity and to avoid confusion let's move forward without explicit scope.

Fixed, removed explicit scope definition.

Reply:
Sorry I missed that part, what I've done in other cases is to have three projects: `core`, `testkit`, `test`. Thus `test` depends on `testkit` but `core` doesn't need to depend on `testkit`
Could that be an option?

Reply:
Sorry, you're absolutely right that we could and should refrain from adding `testkit` as a dependency to the `core`. 
Refactored modules layout as you suggested. Thank you.